health-monitors/gpu-health-monitor/gpu_health_monitor/tests/test_dcgm_watcher/test_dcgm.py

```
@@ -951,3 +955,219 @@ def test_init_monitoring_rolls_back_group_on_failure(self, mock_dcgm_group):
+    before incidents are aggregated per (watch, GPU), so genuine co-occurring
+    incidents on the same GPU and watch are always preserved.
+    """
+
+    A100_PCIE_UNBRIDGED = {
+        "gpu_id": 0,
+        "uuid": "GPU-0",
+        "device_name": "NVIDIA A100 80GB PCIe",
+        "nvlinks": [],
+        "nvlink_link_count": 12,
+        "nvlink_active_link_count": 0,
+    }
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)